### ISSUE TITLE
Integrate with a dummy STT service

### ISSUE THREAD

**Issue by emettely** [bbc/digital-paper-edit-firebase#27]
**Is your feature request related to a problem? Please describe.**
This is the pre-step before we can integrate with Ashley's service being built. Realistically the service won't be built immediately - so we need to have a workaround for now.

**Describe the solution you'd like**
We need to create a fake PSTT RESTful endpoint in the meanwhile. If we don't get around to this and Ashley's service completes, we can invalidate this ticket and start with #28 #29 !

**Describe alternatives you've considered**
We can also build something that uses GCP's STT instead, if all else fails.

**Additional context**

**Comment by emettely** [comment]
Already done 👍
